tools/devservices_healthcheck.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import os
+import subprocess
+from time import sleep
+from typing import List
+
+
+def run_cmd(
+    args: List[str],
```

Comment:
if you add `from __future__ import annotations` this can use new-style `list[str]` without needing to import from `typing`